task0/LinearRegression_Ferhat.ipynb

```diff
@@ -0,0 +1,1853 @@
+{
```

Comment:
<p>Answer: ...</p><p> </p><p>Try to watch standard English rules. You can use Grammarly free version to see some quick tips. </p><p>"Degree polynomial regression fits for data that includes only distance information, but probably there are other variables affect house price noticeably."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bogaziciai/assignment/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>